lib/github_message_factory.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'json'
```

Comment:
Do we need the require here with Zeitwerk loading things now?

Reply:
We do. Zeitwerk only handles loading your application code, not Gems.